docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -1719,13 +1774,18 @@ The decoding algorithm that produces the uncompressed data is as follows:
                read block count using HTREE_BLEN_D and set BLEN_D
             decrement BLEN_D
             compute context ID, CIDD from CLEN
-            read distance code using HTREED[CMAPD[4 * BTYPE_D + CIDD]]
+            read distance code using HTREED[CMAPD[4*BTYPE_D + CIDD]]
             compute distance by distance short code substitution
-         move backwards distance bytes in the uncompressed data and
-            copy CLEN bytes from this position to the uncompressed
-            stream, or look up the static dictionary word, transform
-            the word as directed, and copy the result to the
-            uncompressed stream
+            if distance code is not zero, and distance is not a
```

Comment:
The last two lines should be indented to indicate that these three lines are one instruction:

if distance code is not zero, and distance is not a
   static dictionary reference, push distance to the ring
   buffer of last distances

(and similarly the "move backwards distance ..." and "look up the static dictionary word ..." instructions)